denylist/src/client.rs

```
@@ -0,0 +1,61 @@
+use crate::Result;
+use crate::{error::Error, models::metadata::DenyListMetaData};
+use std::str;
+use std::time::Duration;
+
+/// The default client useragent for denylist http requests
+static DEFAULT_USERAGENT: &str = "https://github.com/helium/miner";
```

This shouldn’t be `miner`

---

It’s also not overridable so calling it default seems misleading